versioned_docs/version-8.0/components/integration-framework/connectors/out-of-the-box-connectors/slack.md

```diff
@@ -80,9 +80,11 @@ A Slack workspace consists of channels in which workspace members can communicat
 
 The **Slack Connector** communicates through a Slack app with a concrete Slack workspace. For example, when sending a Slack message, the message will be posted by the Slack app. For the **Slack Connector** to work, you need to perform the following steps:
 
-1. [Create a Slack app](https://api.slack.com/apps).
-2. [Request required scopes](https://api.slack.com/scopes) - The scopes represent what your app can and cannot do (for example, posting messages). For the Slack Connector to work, at a minimum you need to grant the [`chat:write`](https://api.slack.com/scopes/chat:write) scope.
-3. [Install the Slack app to your workspace](https://api.slack.com/authentication/basics#installing).
+1. You can only install a Slack app to a workspace in which you are a member or that you own. It is not possible if you are guest only [Guide-to-apps-in-Slack](https://slack.com/help/articles/360001537467-Guide-to-apps-in-Slack). 
```

Comment:
Should this read "It is not possible if you have guest-only permissions. See the [guide to apps in Slack](https://slack.com/help/articles/360001537467-Guide-to-apps-in-Slack) for more details."? Or am i misinterpreting here?

Reply:
That is how I read it as well